pkgs/development/libraries/iodash/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config }:
+let
+  version = "0.1.7";
+    /* I pointed it temporarly to my fork to deal with cmake install directives,
+    after the PR is merged it should go back to YukiWorkshop.
+    https://github.com/YukiWorkshop/IODash/pull/2
+    */
+  src  = fetchFromGitHub {
+    owner  = "jappeace";
+    repo   = "IODash";
+    rev    = "095f89d0e088a97d922d429105674cc2e39b2f64";
+    sha256 = "0z5jjn4c01xvaii4w0rp0z5347j6krlwsmwh89dvyaqmb65g7s0j";
+    fetchSubmodules = true;
+  };
+in
+
+stdenv.mkDerivation {
+  pname = "iodash";
+  inherit version src;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "iodash";
  version = "0.1.7";

  src  = fetchFromGitHub {
    owner  = "jappeace";
    repo   = "IODash";
    rev    = "095f89d0e088a97d922d429105674cc2e39b2f64";
    sha256 = "0z5jjn4c01xvaii4w0rp0z5347j6krlwsmwh89dvyaqmb65g7s0j";
    fetchSubmodules = true;
  };
in
```

Also please fetch a patch instead.

---

Yes, I'll make patches for all the libs.